### ISSUE TITLE
For Cycle 9

### ISSUE THREAD

**Issue by wiz** [bisq-network/compensation#470]
##  Summary

BSQ requested: 9100 BSQ @ 0.67 USD = 6097 USD

## Contributions Delivered

Bisq / Bisq Infrastructure [ 8000 BSQ ]
* Completed project to decentralize BSQ explorers, now 6 active
* Reported 2 critical privacy vulnerabilities, patched 1 of them
* Tweaked list of btcnodes, based on latest alerts data
* Upgraded mempool.space to full block explorer

## Roles Performed

* Bisq Markets API Maintainer [ 300 BSQ ]
* Bisq Seednode Maintainer (secondary) [ 100 BSQ ]
* Bisq BTCNode Maintainer (secondary) [ 100 BSQ ]
* Bisq BTCNode Operator (x2) [ 200 BSQ ]
* Bisq Seednode Operator (x2) [ 200 BSQ ]
* Bisq Pricenode Operator [ 100 BSQ ]
* Bisq Analytics Operator [ 100 BSQ ]
* Bisq Twitter @BisqJP [ 0 BSQ ]
* Bisq Keybase Admin [ 0 BSQ ]

## Issues Reported
* bisq-network/bisq#3894
* bisq-network/bisq#3893

## Pull Requests Shipped

* bisq-network/bisq-explorer#24
* bisq-network/bisq-explorer#23
* bisq-network/bisq-explorer#22
* bisq-network/bisq-explorer#21
* bisq-network/bisq-explorer#20
* bisq-network/bisq-explorer#19
* bisq-network/bisq-explorer#18
* bisq-network/bisq-explorer#17
* bisq-network/bisq-explorer#16
* bisq-network/bisq-explorer#15
* bisq-network/bisq-explorer#13
* bisq-network/bisq#3882
* bisq-network/bisq#3880
* bisq-network/bisq#3879
* bisq-network/bisq#3875
* bisq-network/bisq#3874
* bisq-network/bisq#3873
* bisq-network/bisq#3872
* bisq-network/bisq#3864
* bisq-network/bisq#3863
* bisq-network/bisq#3862
* bisq-network/bisq#3859
* bisq-network/bisq#3858
* bisq-network/bisq#3856
* bisq-network/bisq#3855
* bisq-network/bisq#3853
* bisq-network/bisq#3852
* bisq-network/bisq#3849
* bisq-network/bisq#3843
* bisq-network/bisq#3819
* bisq-network/bisq#3815
* bisq-network/bisq#3813

**Comment by chimp1984** [comment]
> Completed project to decentralize BSQ explorers, now 6 active

Such changes should be discussed more up-front. Is it really crucial for Bisq (specially in current low volume situation) to have 6 BSQ explorers? I guess 2-3 should have been enough as well. Those nodes are not crucial from censorship resistant point of view but will add running costs each month now and added extra setup costs for this cycle.

**Comment by wiz** [comment]
> Such changes should be discussed more up-front. Is it really crucial for Bisq (specially in current low volume situation) to have 6 BSQ explorers?

@chimp1984 Well, my compensation request above doesn't include any BSQ explorer hosting charges, only the development work, but the BSQ explorer roles were expanded to 2 maintainers and 4 operators as part of [DAO proposal 143 to decentralize BSQ explorers](https://github.com/bisq-network/proposals/issues/143), which I believe you approved.

But I do appreciate Bisq's current financial situation and to help support the project I'd be happy to waive my hosting fee for the BSQ explorer going forward, as it doesn't use much server/network resources. I will also consider how to cut other infrastructure costs.

<img width="682" alt="Screen Shot 2020-01-15 at 8 00 01" src="https://user-images.githubusercontent.com/232186/72390290-b2dcd000-376d-11ea-90e9-897840d60280.png">

**Comment by chimp1984** [comment]
@wiz Oh sorry, yes forgot that there was a proposal.... got too heated with cutting costs ;-)

**Comment by sqrrm** [comment]
I think the work on the explorer distribution will pay for itself quite quickly as it's now a much easier setup and maintenance task, and as said, it was the result of an approved proposal. I used to spend time on the explorer which I now think will be reduced to 0.

**Comment by cbeams** [comment]
> * Bisq Markets API Maintainer [ 300 BSQ ]
> * Bisq Seednode Maintainer (secondary) [ 100 BSQ ]
> * Bisq BTCNode Maintainer (secondary) [ 100 BSQ ]
> * Bisq BTCNode Operator (x2) [ 200 BSQ ]

Was it an oversight, @wiz, that these line items didn't link to their respective role issues? I assume so, as I see you've consistently done so in previous cycles. I see that Cycle 9 updates are missing on those roles as well (at least the first two that I checked: seednode maintainer and markets website+api maintainer).

Note to all, @wiz and I discussed his roles, and the meaning of 'maintainer' has drifted in a number of cases here (and elsewhere). For cycle 10, we'll get most of these roles reassigned (and in some cases redefined) so that everything makes sense. Wiz will of course continue to play his operator roles.
